roles/openshift_health_checker/openshift_checks/etcd_imagedata_size.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about an etcd cluster size based on OpenShift image data.
+"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdImageDataSize(OpenShiftCheck):
+    """Check that total size of OpenShift image data does not exceed the recommended limit in an etcd cluster"""
+
+    name = "etcd_imagedata_size"
+    tags = ["etcd"]
+
+    def run(self, tmp, task_vars):
+        ansible_mounts = get_var(task_vars, "ansible_mounts")
+
+        etcd_mountpath = self.get_etcd_mountpath(ansible_mounts)
+        etcd_avail_diskspace = etcd_mountpath["size_available"]
+        etcd_total_diskspace = etcd_mountpath["size_total"]
+
+        etcd_imagedata_size_limit = get_var(task_vars,
+                                            "cli_etcd_max_image_data_size_bytes",
```

Comment:
@rhcarvalho per the discussion in https://github.com/openshift/openshift-ansible/commit/d60ac18a64c4fd33e3a7c8d0fb3ce94f79dfa742#commitcomment-21856259, I have updated this to be a bit more intuitive, however will still try to get feedback for a final variable name